2_feature_importance/2_feature_importance.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
_Commented on notebook `2_feature_importance/2_feature_importance.ipynb` Cell 1 Line 4_
```markdown
<a href="https://colab.research.google.com/github/oriol-pomarol/codegeo_workshops/blob/main/2_feature_importance/2_feature_importance.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Feature importance
During this workshop we will explore three different ways to estimate the importance of our input features on the model outputs: impurity feature importance, permutation feature importance and shap feature importance.
```
I would go a bit more in detail into what feature importance means and why it is useful (if not here, maybe at the end). I understand you are going to explain that in person, but in general try to make the notebook standalone, as many people may not have access to the recording.

_Respond and view the context [here](https://gitnotebooks.com/oriol-pomarol/codegeo_workshops/pull/1)._

Reply:
This is part of the presentation